### PR TITLE
Changelog automation

### PR DESCRIPTION
**Pull request by ardier16**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,8 +14,9 @@
     "lint:styles": "stylelint \"src/**/*.{css,sass,scss}\" --max-warnings=0",
     "lint:scripts": "eslint \"{src,config}/**/*.{js,ts,jsx,tsx}\" --cache --fix --max-warnings=0 && tsc --noEmit",
     "preview": "vite preview",
-    "rsc": "node scripts/release-sanity-check.mjs",
-    "generate-ether-types": "typechain --target=ethers-v5 'abis/**/*.json' --out-dir src/types/contracts"
+    "generate-ether-types": "typechain --target=ethers-v5 'abis/**/*.json' --out-dir src/types/contracts",
+    "package-version": "node -p \"require('./package.json').version\"",
+    "version": "rlx changelog release $(yarn package-version) && git add ."
```
Comment (ihordiachenko) [inline package.json:19]: Would be nice to put it into the `preversion` hook but I guess it won't fit into the current CI workflow

**Review by ihordiachenko — APPROVED** [review]

File: .gitignore [hunk 2]
```diff
@@ -23,8 +23,8 @@ deprecated
 *.sln
 *.local
 stats.html
-.yarn
-.yarnrc.yml
+.yarn/*
+!.yarn/releases
```
Comment (napalmpapalam) [inline .gitignore:27]: https://yarnpkg.com/getting-started/qa#which-files-should-be-gitignored

**Review by napalmpapalam — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (ardier16) [reply package.json:19]: @ihordiachenko the package.json version is not bumped yet when the `preversion` hook is called. And you still need to stage changed files using 'git add`: https://docs.npmjs.com/cli/v9/commands/npm-version#description

File: .gitignore (hunk 2 above)
Comment (ardier16) [reply .gitignore]: updated

File: package.json (hunk 1 above)
Comment (ihordiachenko) [reply package.json:19]: Yeah, you're right. Re-checked the shelf stuff and it also uses the version hook

**Review by napalmpapalam — APPROVED** [review]

**Pull request merged**